sql/analyzer/fix_exec_indexes.go

```diff
@@ -388,6 +387,24 @@ func (s *idxScope) finalizeSelf(n sql.Node) (sql.Node, error) {
 		if nc, ok := ret.(sql.CheckConstraintNode); ok && s.checks != nil {
 			ret = nc.WithChecks(s.checks)
 		}
+		if jn, ok := ret.(*plan.JoinNode); ok {
+			if len(s.parentScopes) == 0 {
```

Comment:
if there were >1 parent, would we summate all the parent offsets?

Reply:
what happened to the HashLookup special case? does that one not matter?

Reply:
as far as I can tell, we no longer have to prevent StripRowNodes from being added over HashLookups